### ISSUE TITLE
Routing not working on creation of new project

### ISSUE THREAD

**Issue by arunrreddy** [vuejs/vue-cli#5792]
### Version
4.5.3

### Environment info
```
  System:
    OS: Linux 4.15 Ubuntu 18.04.5 LTS (Bionic Beaver)
    CPU: (4) x64 Intel(R) Core(TM) i5-6200U CPU @ 2.30GHz
  Binaries:
    Node: 10.15.1 - ~/.nvm/versions/node/v10.15.1/bin/node
    Yarn: 1.21.1 - /usr/bin/yarn
    npm: 6.12.0 - ~/.nvm/versions/node/v10.15.1/bin/npm
  Browsers:
    Chrome: 83.0.4103.97
    Firefox: 79.0
```

### Steps to reproduce
create a new project with vue-cli (4). Select manual and select vue 3.0 with all other options. Run the application and route to the about page. The about page content is never shown

### What is expected?
About page content should be shown while routing to /about.

### What is actually happening?
It is staying the home page but only the URL changes. This happens with both history mode and hash mode.

**Comment by Gadrawingz** [comment]
Going project

**Comment by sodatea** [comment]
Fixed in 4.5.4
